doc/flame/other/util.md

```diff
@@ -162,7 +162,7 @@ Methods:
  - `darken`: Darken the shade of the color by an amount between 0 to 1.
  - `brighten`: Brighten the shade of the color by an amount between 0 to 1.
 
-Factories:
+Factories (available on `ColorExtension`):
```

Comment:
I think this is still not completely clear that you have to write `ColorExtension.*`, maybe just spell it out in the list? Like `ColorExtension.fromRGBHexString`

Reply:
Consider it done